src/python/circllhist/__init__.py

```diff
@@ -0,0 +1,77 @@
+import circllhist.ffi as ffi
+
+
+class circllbin:
```

Comment:
Explicitly inherit from `object` to always use "new style classes" for Python 2 and 3 consistency. 

And use `__slots__`.